data.sql

```diff
@@ -0,0 +1,8 @@
+\d animals;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-2-3',0,true,10.23 );
+SELECT * FROM animals;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15',2,true,8 ); 
+SELECT * FROM animals;
+UPDATE animals SET neutered = false WHERE id = 1;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-1-7',1,false,15.04 ), (4, 'Devimon', '2017-5-7', 5, true,11);
```

Comment:
Good work creating animals! 💯 

As there's a lot of information that you need to enter manually, could happen that you have some typos. For example here, the animal `Devimon` has a different date that we have to input. 

- Kindly fix the date_of_birth of Devimon, the correct one is `2017/05/12`
- Kindly remember to use spaces before a `,` (coma) so your code will be more ordered and readable 🥇